tools/scripts/CaptureDevHomeLogs.ps1

```diff
@@ -0,0 +1,18 @@
+# Quick script to capture log files and create a zip file for upload to GitHub
+# Currently this only creates one based on Date. We could add a time, so that 
+# the ISV can have more than one Zip file
+
+$MyZip=$(Get-Date -Format yyyy-MM-dd) + "DevHome.Zip"
+$LogPath="C:\Users\" + $Env:UserName  + "\AppData\Local\Packages"
+$LogFiles = Get-ChildItem -Path $LogPath -Filter "*.dhlog" -recurse
+$Temp = [System.IO.Path]::GetTempPath()
+$TempFile=$Temp+"\"+$MyZip
+

```

Comment:
```suggestion
Get-Process *DevHome* -ErrorAction Ignore | Stop-Process

```
This will stop DevHome and the extensions so there will not be file-in-use errors during archiving. I have tested this locally.

Reply:
Should we add some sort of "This will exit Dev Home, are you sure?" check? I'm thinking the flow here is a person opens an issue, we ask for logs, they may or may not be running Dev Home some time later when they run this script, so we have no idea what they might be in the middle of. Although we can warn them for now in the ask.